src/agencies-context.ts

```diff
@@ -0,0 +1,14 @@
+import { createContext } from "preact";
```

Comment:
Since we'll possibly have more context, maybe an opportunity for some folder organization? As `src/context/agencies-context.ts` for example.